package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "metal-myComponent",
+	"version": "0.0.0",
+	"description": "My awesome Metal.js project",
+	"license": "BSD",
+	"repository": "zxdgoal/metal-myComponent",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "src/MyComponent.js",
+	"files": [
+		"src",
+		"test"
+	],
+	"scripts": {
+		"test": "gulp test"
+	},
+	"keywords": [
+		"metal"
+	],
+	"dependencies": {
+    "bootstrap": "^3.3.6",
+		"closure-templates": "20140422.0.0",
+		"metal": "metal/metal.js#master",
+		"metal-component": "metal/metal-component#master",
+		"metal-dom": "metal/metal-dom#master",
+		"metal-events": "metal/metal-events#master",
+		"metal-soy": "metal/metal-soy#master"
+	},
+	"devDependencies": {
+		"gulp": "^3.8.11",
```

Comment:
You'll also need to add `babel-cli` and `babel-preset-metal` dev dependencies here, just like in [metal-modal's package.json](https://github.com/metal/metal-modal/blob/master/package.json#L39)